metrics/network/cpu_statistics_iperf.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Description of the test:
+# This test runs the 'iperf3' network benchmark and
+# we retrieve cpu information while running the networking
+# test with 'perf'.
+
+set -e
+
+# General env
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../lib/common.bash"
+
+TEST_NAME="cpu information"
+
+# Image name
+image="networkstatic/iperf3"
```

Comment:
Ooh, that is a different image than we have used for all our other previous tests (which was gabyct/networking iirc).
That's great, that it is a smaller image (just has iperf3 in it) - but... what I think I'd really like is for us to have all the test images we can (where practical) embedded into our source tree - so, if we could add a local Dockerfile and build method like we have over in the [blogbench](https://github.com/kata-containers/tests/tree/master/metrics/storage) test, that'd be great.
As jodh would say - you get 'bonus points' if you can make that Dockerfile 'versioned', so we don't have to rebuild it every time if we detect the correct version of the image is already installed in Docker...

Extra notes on that then - adding Dockerfiles increases our CI time, as we then have to build them all from sources. And as our CIs run on 'clean instances', we have to do that for every run - we don't (presently) cache the images.
/cc @chavafg

Reply:
aha, /me notes #404 - so seems we can move to using that as well ;-)